frontend/src/components/worksheets/items/TableItem/TableItem.js

```diff
@@ -328,6 +333,11 @@ class _TableContainer extends React.Component {
 const styles = (theme) => ({
     tableContainer: {
         position: 'relative',
+        '&::-webkit-scrollbar': {
```

Comment:
Why do this instead of `overflow: hidden`?

Reply:
I just switched the method! Thanks for your advice.

Reply:
Is there a chance we might cut off text in the table, this way? Or is that not going to be an issue?

Reply:
I think you can scroll left and right to see the text if it is overflow. In the gif you can see no text is cut off.

Reply:
<img width="956" alt="Screen Shot 2020-09-30 at 2 07 52 PM" src="https://user-images.githubusercontent.com/34461466/94739848-731f3e00-0326-11eb-8299-6bf4e446a69e.png">

Reply:
It is proved that the `overflow` method will cut off the text. So I switched back to the specific webkit-scrollbar method.